app/models/post.js

```diff
@@ -5,5 +5,5 @@ export default Model.extend({
   content: attr('string'),
   createdAt: attr('string'),
   category: belongsTo('category'),
-  // user: belongsTo('user', { async: true }),
+  user: belongsTo('user', { async: true }),
```

Comment:
Why do we need to have async here? Is it a way to prioritize/deprioritize the data flow?

Reply:
@Agathebadia _"Ember Data resolves sync relationships with the related resources available in its local store, hence it is expected these resources to be loaded before or along-side the primary resource."_   https://api.emberjs.com/ember-data/release/functions/@ember-data%2Fmodel/belongsTo

The default behaviour is `async: true`. This means that Ember should get the relationship resource by making an API call to the BE. If `async` is set to `false`, then Ember will only try to get it from its store (assuming it was already pre-loaded). If it does not exist, then it will return undefined.

I'm happy to demonstrate this in our next session

Reply:
Ooooh I see, thanks a lot Ronnie!! 🙏